go/common/flag/flag.go

```diff
@@ -0,0 +1,168 @@
+package flag
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+)
+
+// WrappedFlag is a construct that allows to have go flags while obeying to a new set of restrictiveness rules
+type WrappedFlag struct {
+	flagType     string
+	ptr          any
+	defaultValue any
+	description  string
+}
+
+// GetString returns the flag current value cast to string
+func (f WrappedFlag) GetString() string {
```

Comment:
shouldn't it be a pointer: `(f *WrappedFlag)`

Reply:
No, there's no modification in the instance, so it should be ok.